uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -195,6 +197,17 @@ define([
     }
 
     /**
+     * Initialises watcher for accrue value
+     */
+    function initAccrueValueWatcher () {
+      if (vm.isMode('view')) { return; }
+
+      $rootScope.$watch(
+        function () { return vm.request.toil_to_accrue; },
```

Comment:
is this covered by the tests?

Reply:
It doesn't and it doesn't seem to be straightforward. When I add a test in request.controller it doesn't call the TOIL controller. This piece of code was manually tested.